### ISSUE TITLE
The application won't open in Mac OSX

### ISSUE THREAD

**Issue by jgongo** [toggl/toggldesktop#2379]
The application stopped working on macOS Sierra 10.12.6. When starting the application using Spotlight I get the following error message:

> The application “TogglDesktop” can’t be opened.

If I try to double click the icon in the Applications folder, the application simply won't start without providing any feedback.

After encountering this error I removed and reinstalled the application, and I was able to run it again. But then after closing it I get this error again. If I try to open the application from the command line I get the following:

```
$ open /Applications/TogglDesktop.app
LSOpenURLsWithRole() failed with error -10810 for the file /Applications/TogglDesktop.app.
```

[Clearing the local data](https://support.toggl.com/clearing-the-local-data-from-toggl-desktop/) does not help.

**Comment by IndrekV** [comment]
Same as #2282. Are you also using the AppStore version. If so I suggest you try out the version from our website. The website version is always more up to date than AppStore version.

**Comment by edjere** [comment]
I had the same issue this morning. All of a sudden the app did not open, and I did not even get a prompt or message.
I uninstalled it and after unsuccessful installation attempts from the AppStore I was able to fix it by downloading this version: https://support.toggl.com/toggl-desktop-for-mac-osx/

Hope it helps.

**Comment by afroald** [comment]
I was not using the app store version. I was on the beta release channel so the app auto-updated to 7.4.91 or so which won't start.

So how I fixed it:
1. Install 7.4.84 from the website
2. Go to about, set release channel to stable (in the mean time the app already had the update downloaded)
3. Close app
4. Install 7.4.84 again to rollback the update from the dev release channel

**Comment by IndrekV** [comment]
Thank you for the feedback. We'll review the current beta version.

**Comment by goto10** [comment]
Going back to 7.4.84 got that app running again. 
Here is what the Crash Report from the newer version is complaining about: 
Library not loaded: /usr/local/lib/libcrypto.1.1.dylib

**Comment by IndrekV** [comment]
This issue is fixed in the latest mac beta (7.4.111)

Thank you everybody for sharing the details.
